src/js/views/patients/sidebar/flow/flow-sidebar_views.js

```diff
@@ -98,13 +117,23 @@ const SidebarView = View.extend({
   showFlow() {
     this.canEdit = this.model.canEdit();
 
+    this.showName();
```

Comment:
I don't think name or details need to be in this function..  they can just be in onRender.. this function appears to be needed if something changes that affects `canEdit`

Reply:
really I think this method did more a bit ago and it likely doesn't make any sense now..  `this.canEdit = this.model.canEdit();` should move into `showActions` and  changing the owner should do that.

Reply:
I moved `showName()` and `showDetails()` up to the render function. And moved the `this.canEdit = ...` to `showActions()`.